src/QMCWaveFunctions/detail/CUDA/matrix_update_helper.cu

```diff
@@ -370,15 +372,17 @@ __global__ void add_delay_list_save_sigma_VGL_kernel(int* const delay_list[],
         Urow_iw[col_id] = T(0);
     }
 
+    // Set y to zero
     T* __restrict__ binv_iw          = binv[iw];
     const int num_blocks_delay_count = (delay_count + COLBS - 1) / COLBS;
     for (int ib = 0; ib < num_blocks_delay_count; ib++)
     {
       const int col_id = ib * COLBS + tid;
       if (col_id < delay_count)
-        binv_iw[delay_count * binv_lda + col_id] = binv_iw[delay_count + binv_lda * col_id] = T(0);
+        binv_iw[delay_count + binv_lda * col_id] = T(0);
     }
 
+    // Set x to zero
```

Comment:
Seems like this copy is a typo, based on the cpu version and old comment x should get set to 1 assuming
```
x = binv_iw[delay_count * binv_lda + delay_count]
```

Reply:
Seems so. Let me fix it.